server/src/main/java/org/elasticsearch/action/admin/indices/close/CloseIndexClusterStateUpdateRequest.java

```diff
@@ -25,7 +25,17 @@
  */
 public class CloseIndexClusterStateUpdateRequest extends IndicesClusterStateUpdateRequest<CloseIndexClusterStateUpdateRequest> {
 
+    private long taskId;
+
     public CloseIndexClusterStateUpdateRequest() {
```

Comment:
make taskId a mandatory parameter by requiring it in the constructor and making the field final?